### PR TITLE
feat: enabled font size, closes #83

### LINKED ISSUE #83: Font sizes are getting ignored
Hi, I just started to use this package and I think it's really cool. Here's one problem I'm having, no matter whatever the font I'm setting in the sheet it's getting ignored. I also tried with markdown syntax like `#` but it's getting ignored as well. 

```json
    "gatsby-source-google-docs": "^2.0.0-beta.26",
    "gatsby-transformer-remark": "^2.8.42",
    "gatsby-transformer-sharp": "^2.5.7",
    "react": "^16.12.0",
    "gatsby": "^2.23.12",
```

### PR DESCRIPTION
**Pull request by somoy99**
[:heavy_check_mark:] added `enableFontSize` to enable font size

### CODE REVIEW

**Comment by cedricdelpoux** [comment]
You need to use the font size used in google Docs. "font-size: 12px" was just an example

**Comment by somoy99** [comment]
> You need to use the font size used in google Docs. "font-size: 12px" was just an example

:rofl: ahh.. sorry. so, seems like I need to look up a bit more.

File: utils/google-document.js [hunk 1]
```diff
@@ -122,7 +124,13 @@ class GoogleDocument {
     text = text.replace(/_/g, "\\_") // Prevent _ to be italic
 
     if (baselineOffset === "SUPERSCRIPT") {
-      text = `<sup>${text}</sup>`
+      if (this.enableFontSize) {
+        text = `<span style="font-size:${
+          fontSize.magnitude
```
Comment (somoy99) [inline utils/google-document.js:129]: something like this would work?

Comment (cedricdelpoux) [reply utils/google-document.js:129]: You must do not put this inside `if (baselineOffset === "SUPERSCRIPT") {` 
It's a different transformation

**Review by cedricdelpoux — CHANGES_REQUESTED** [review]

**Comment by cedricdelpoux** [comment]
Thank you for your work @somoy99. I made some adjustments to reach 100% code coverage. I merged your commit in the PR #85.

Finally, I found a way to add font-size feature without the `enableFontSize` option. I used `em` unit relative to normal text font size. I could be much better using it with gatsby sites.

**Pull request closed without merging**